test/features/deploy_kogito_runtime.feature

```diff
@@ -32,6 +32,47 @@ Feature: Deploy Kogito Runtime
       | runtime    | example-service         | native   |
       | quarkus    | process-quarkus-example | enabled  |
 
+#####
+
+  Scenario Outline: Deploy DMN <example-service> with native <native> using Kogito Runtime
+    Given Kogito Operator is deployed
+    And Clone Kogito examples into local directory
+    And Local example service "<example-service>" is built by Maven and deployed to runtime registry with Maven configuration:
+      | native | <native> |
+
+    When Deploy <runtime> example service "<example-service>" from runtime registry
+
+    Then Kogito Runtime "<example-service>" has 1 pods running within 10 minutes
+    And HTTP POST request on service "<example-service>" is successful within 2 minutes with path "Traffic Violation" and body:
+      """json
+      {
+          "Driver":{"Points":2},
+          "Violation":{
+              "Type":"speed",
+              "Actual Speed":120,
+              "Speed Limit":100
+          }
+      }
+      """
+
+    @rhpam
+    @springboot
+    Examples:
+      | runtime    | example-service        | native   |
+      | springboot | dmn-springboot-example | disabled |
+
+    @rhpam
+    @quarkus
```

Comment:
dmn-quarkus-example

Reply:
No, the smoke test is meant to check operator functionality.
This test is checking DMN function rather than operator, so no need to have it as smoke.